programs/drift/src/math/orders.rs

```diff
@@ -576,6 +576,13 @@ pub fn get_max_fill_amounts(
 fn get_max_fill_amounts_for_market(user: &User, market: &SpotMarket) -> DriftResult<u128> {
     let position_index = user.get_spot_position_index(market.market_index)?;
     let token_amount = user.spot_positions[position_index].get_signed_token_amount(market)?;
+    get_max_withdraw_for_market_with_token_amount(token_amount, market)
+}
+
+pub fn get_max_withdraw_for_market_with_token_amount(
```

Comment:
might just add #inline[always] to this just so it doesn't make `get_max_fill_amounts_for_market` any more expensive